src/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>TV maize movies and series</title>
+  </head>
+  <body>
+    <div class="popUpBox" id="popUpBox"></div>
+    <header>
+      <nav>
```

Comment:
Please design `Home Page Header` according to [this](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) mock-up. There should be a logo and navigation links displayed horizontally.